source/usage-examples.txt

```diff
@@ -120,3 +120,4 @@ Available Usage Examples
 - :doc:`Run a Command </usage-examples/command>`
 - :doc:`Watch for Changes </usage-examples/changeStream>`
 - :doc:`Perform Bulk Operations </usage-examples/bulkWrite>`
+- :doc:`Perform a Convenient Transaction </usage-examples/convenient-transactions>`
```

Comment:
Question:
I think the name of the API is "Convenient Transaction API", but does that mean that using it to complete a transaction is a "convenient transaction"? If not, maybe this could be "Use the Convenient Transaction API" instead.

Reply:
I wasn't able to find it in the [spec](https://github.com/mongodb/docs-node/pull/751/files#r1307624029), so I think it would be better to describe as mentioned.

Reply:
Convenient Transaction API is the terminology that Neal provided, but I agree that its still a regular transaction. I think the Usage example can just be called Perform a transaction.